.github/workflows/upload.yaml

```
@@ -0,0 +1,56 @@
+        uses: unfor19/install-aws-cli-action@v1
+        with:
+            version: 2                         # default
+            verbose: true                     # default
+            arch: amd64                        # allowed values: amd64, arm64
+
+    
+      - name: Checkout
+        uses: actions/checkout@main
+
+      - name: Setup AWS CLI
+        uses: aws-actions/configure-aws-credentials@v1
```

actionlint (1.7.4)

the runner of "aws-actions/configure-aws-credentials@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)